distributions/aws/infra_configs/scripts/aws/acm.py

```diff
@@ -0,0 +1,152 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import boto3
+import logging
+import time
+
+from botocore.exceptions import ClientError, WaiterError
+from .route53 import Route53HostedZone
+from typing import Any
+
+logger = logging.getLogger(__name__)
+
+
+class AcmCertificate:
+    """
+    Encapsulates ACM functions.
+    """
+
+    def __init__(
+        self,
+        domain: str,
+        hosted_zone: Route53HostedZone = None,
+        region: str = "us-east-1",
```

Comment:
why hard coded region? Certificate has to be in us-east-1 ?

Reply:
or it is default?

Reply:
default